_config.yml

```diff
@@ -21,6 +21,8 @@ kramdown:
 collections:
   docs:
     output: true
+  sponsors:
+    output: false
```

Comment:
With a collection we can just manage a bunch of small files in the `_sponsors` directory to add/remove sponsors.  For now, they are sorted by name, but if we want to sort them further this seems to be supported:
https://jekyllrb.com/docs/collections/#custom-sorting-of-documents